src/main/java/org/broadinstitute/hellbender/utils/GenomeLocParser.java

```diff
@@ -393,7 +393,17 @@ public GenomeLoc createGenomeLoc( final GATKRead read ) {
      * @return
      */
     public GenomeLoc createGenomeLoc(final Feature feature) {
-        return createGenomeLoc(feature.getContig(), feature.getStart(), feature.getEnd());
+        return createGenomeLoc(feature.getContig(), feature.getStart(), feature.getEnd(), true);
+    }
+
+    /**
+     * Creates a GenomeLoc from a Tribble feature
+     * @param feature
+     * @param skipValidation  true to disable check for features to be on the reference
+     * @return
+     */
+    public GenomeLoc createGenomeLoc(final Feature feature, final boolean skipValidation) {
+        return createGenomeLoc(feature.getContig(), feature.getStart(), feature.getEnd(), !skipValidation);
```

Comment:
It is very confusing to introduce a new boolean parameter `skipValidation` that is the inverse of the existing `mustBeOnReference` boolean that is everywhere else in this class. Can you rename this one to `mustBeOnReference` as well for consistency, and remove the negation / flip the values passed in?